tests/atomistics/structure/test_analyse.py

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+import unittest
+import numpy as np
+from pyiron.atomistics.structure.atoms import Atoms, CrystalStructure
+
+
+class TestAtoms(unittest.TestCase):
+    def test_get_layers(self):
```

Comment:
Please break the tests into smaller functions and add either docstrings or short messages to the asserts.  This helps a lot if anyone else has to debug the tests.